.github/workflows/release-please.yml

```
@@ -0,0 +1,97 @@
+on:
+  push:
+    branches:
+      - main
+
```

If you don't specify permission, it defaults to very loose ones
```suggestion

permissions:
  contents: read

```